docs/tools/developer-tools/jupyter-notebooks.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Jupyter Notebooks
+description: Experiment with contracts in Jupyter Notebooks
+sidebar_label: Jupyter Notebooks
+sidebar_position: 71
+---
+
+# Jupyter Notebooks
+
+Jupyter Notebooks are a document format that can include code, text, and other rich output. And with the appropriate setup they support Soroban Rust contracts.
+
+To use Soroban contracts in a Jupyter Notebook, the following setup is required. The following setup uses Visual Studio Code, but any Jupyter client and server can be used.
+
+:::caution 
```

Comment:
I think the Caution box isn't set up correctly. The preview has the entire article in the Caution box!

Reply:
Yes, it needs an empty line after the opening admonition tag, and before the closing tag. like this

```markdown
:::caution

Rust support in Jupyter ...

:::
```